tls/s2n_x509_validator.c

```diff
@@ -649,7 +632,20 @@ static S2N_RESULT s2n_x509_validator_verify_cert_chain(struct s2n_x509_validator
      * only called if time validation is enabled.
      */
     if (conn->config->disable_x509_time_validation) {
-        RESULT_GUARD(s2n_x509_validator_disable_time_validation(conn, validator));
+        /* Setting an X509_STORE verify callback is not recommended with AWS-LC:
+         * https://github.com/aws/aws-lc/blob/aa90e509f2e940916fbe9fdd469a4c90c51824f6/include/openssl/x509.h#L2980-L2990
```

Comment:
Isn't this comment in the wrong place, and no longer accurate? Why is the new callback safe too?